wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -1,27 +1,81 @@
 {% load markdown_deux_tags %}
+{% load humanize %}
+{% load static %}
+
 {% if object.metadata.cancelled_election.title %}
     <h2>{{ object.metadata.cancelled_election.title }}</h2>
 {% else %}
-    <h2>Cancelled Election</h2>
+    <div>
+        {% if object.winner_count == object.people.count %}
+            <b>Uncontested Election</b>
+
+            <p>This election was uncontested because the number of candidates who stood was equal to the number of available seats.
+                There {{ object.winner_count|pluralize:"is,are" }} {{ object.winner_count|apnumber }}
+                seat{{ object.winner_count|pluralize }} in this ward/division, and only
+                {{ object.people.count|apnumber }} candidate{{ object.people|pluralize }}.</p>
+
+            <p>No votes will be cast, and the candidate{{ object.people|pluralize }} below have been automatically declared
+                the winner{{ object.people|pluralize }}.</p>
+
+            {% if object.display_as_party_list %}
+                {% include "elections/includes/_people_list_with_lists.html" with people=object.people %}
+            {% else %}
+                {% include "elections/includes/_people_list.html" with people=object.people %}
+            {% endif %}
+
+        {% else %}
+            {% if object.people.count >= 1 %}
```

Comment:
I think this can just be `if object.people`? But just to make sure, is this only shown when we've set the cancelled flag?